### PR TITLE
DOMA-4288 added organization and role name to user info for UseDeskWidget

### PR DESCRIPTION
**Pull request by VKislov**
added 2 more fields, role name and organization name

### CODE REVIEW

File: apps/condo/domains/common/components/UseDeskWidget.tsx [hunk 1]
```diff
@@ -4,7 +4,11 @@ import getConfig from 'next/config'
 import get from 'lodash/get'
 
 const { publicRuntimeConfig:{ UseDeskWidgetId } } = getConfig()
-const fieldId = 20560
+const fieldId = {
```
Comment (AleX83Xpert) [inline apps/condo/domains/common/components/UseDeskWidget.tsx:7]: What about the more understandable name?
```suggestion
const useDeskFieldsIdsMap = {
```

**Review by AleX83Xpert — APPROVED** [review]

**Review by DanilovMaxim13 — APPROVED** [review]

File: apps/condo/domains/common/components/UseDeskWidget.tsx (hunk 1 above)
Comment (VKislov) [reply apps/condo/domains/common/components/UseDeskWidget.tsx:7]: yes you are right, I forgot to change :)

File: apps/condo/domains/common/components/UseDeskWidget.tsx [hunk 2]
```diff
@@ -4,7 +4,11 @@ import getConfig from 'next/config'
 import get from 'lodash/get'
 
 const { publicRuntimeConfig:{ UseDeskWidgetId } } = getConfig()
-const fieldId = 20560
+const useDeskFieldsIdsMap = {
+    tin: 20560,
```
Comment (akarjakin) [inline apps/condo/domains/common/components/UseDeskWidget.tsx:8]: Better to put such data to .helm, in order to be able to change values without release process.

Comment (VKislov) [reply apps/condo/domains/common/components/UseDeskWidget.tsx:8]: okay, I'll move it to helm when I cut out the jivosite id from there in the next release

**Pull request merged**